spec/helpers/leaderboard_helper_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe LeaderboardHelper, type: :helper do
+  describe "#participation_points_active?" do
+    context "when points are awarded for participation" do
+      before { create(:rule, key: :points_for_participation, value: 1) }
+
+      it { expect(helper.participation_points_active?).to eq(true) }
+    end
+
+    context "when points are not awarded for participation" do
+      it { expect(helper.participation_points_active?).to eq(false) }
+    end
```

Comment:
Maybe use the `be_falsey` / `be_truthy` matchers instead of calling the `eq` matcher with an explicit argument? Just a style thing, no biggie.